stacks/values.yaml

```diff
@@ -0,0 +1,38 @@
+# Visit https://github.com/okteto/charts/tree/master/stacks#reference for YAML reference
```

Comment:
Maybe addd a second folder there with samples?

Reply:
Do you think we should instead have a okteto/docs page so it's more formal?

Reply:
Yes, let move parts of this to the docs. This is just a temporal link